open-company/open-company-web
# Show private or public icons besides section name on mobile.

Card: https://trello.com/c/fANGeo0Y
Item:
> *Update the privacy label (mobile only)

Design:
https://share.goabstract.com/1ac1f9a8-7d12-4771-a02f-78be9cd670a2
https://share.goabstract.com/025c1084-91e0-4b15-bf21-a29efa5089f8

To test:
- open mobile app
- open a public section
- [x] do you see the word icon besides the section name? Good
- [x] check on desktop that the world icon is still light grey and green when selected
- open a private section
- [x] check as above
